src/schema/v2/partner/partner.ts

```diff
@@ -752,6 +753,41 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
         type: GraphQLString,
         resolve: ({ collecting_institution }) => collecting_institution,
       },
+      contactsConnection: {
+        description: "A connection of contacts from a Partner.",
+        type: contactsConnection.connectionType,
+        args: pageable({
+          page: {
```

Comment:
Likely don't need these arguments, can just use `pageable({})` for the standard connection-style arguments. These are mostly used to support pagination where you may want to jump to page 5 on an initial load. Not sure the UI for this, but you _prob_ dont need it.

Reply:
Fair points! Yeah I just want to be able to paginate through a list of contacts when needed in volt, will simplify

Reply:
should be adjusted!